pkg/manifest/manifest.go

```diff
@@ -47,6 +47,10 @@ type Manifest struct {
 	// and isn't meant to be user-specified
 	BundleTag string `yaml:"-"`
```

Comment:
Do we use this field, `BundleTag`, to hold the final computed bundle reference? Or can we remove it?

Reply:
I see it used in code but it's super confusing and I think a relic from a previous config setting we removed. Now may be a real good time to get rid of it completely (in a follow-up).

Reply:
> Do we use this field, BundleTag, to hold the final computed bundle reference? Or can we remove it?

Not anymore - now we use `Reference` to track the bundle reference.  Indeed, `BundleTag` is ripe for removal.  I can create an issue.